components/FullSizeMusicPlayer/PlayerControls.tsx

```diff
@@ -101,8 +101,7 @@ export const PlayerControls = () => {
 
 const styles = StyleSheet.create({
   monospacedText: {
-    fontFamily: Platform.OS === "ios" ? "Courier New" : "monospace",
-    letterSpacing: -1,
-    fontWeight: "bold",
+    fontFamily: "Helvetica",
```

Comment:
i don't think `Helvetica` will work on Android. do you want to just leave in `monospace` for Android for now? https://stackoverflow.com/questions/60619440/cant-change-font-family-react-native#:~:text=If%20you%20are%20using%20Android,need%20to%20use%20custom%20font.&text=First%20you%20need%20to%20add,ttf%20file%20for%20it.

Reply:
if you wanna just kick that can down the road, remove the `Platform` import cause it's not used anywhere else in the file.

Reply:
Just tried testing on Android, Roboto works well on that OS so let's update to that